contracts/batcher/Batcher.sol

```diff
@@ -51,7 +51,7 @@ abstract contract Batcher is IBatcher, UOwnable {
         USDC.push(to, amount);
     }
 
-    function rebalance() external {
+    function rebalance() public {
```

Comment:
why is this public?

Reply:
This is so we can call `rebalance` from `repayUSDC`